lib/src/query_component.dart

```diff
@@ -15,20 +15,31 @@ import 'reporting_query_parser.dart';
            placeholder="Advertiser ID: 164337" debugId="advertiser-id-input">
     <input [(ngModel)]="insertionOrderId" 
            placeholder="Insertion Order ID: 8127549" debugId="io-id-input">
+    <br>
+    <input type="checkbox" [(ngModel)]="highlightUnderpacing"
+           debugId="underpacing" name="underpacing">
+    <label for="underpacing">Highlight underpacing insertion orders</label><br>
+    <input *ngIf="highlightUnderpacing" [(ngModel)]="underpacingThreshold"
```

Comment:
this is an interesting way to do it.
an alternative would be to have it as a cell in excel. (then a user can change it and it will automatically recalculate. does that work here?)